services/data_support/data_rag.py

```
@@ -0,0 +1,62 @@
+import vanna
+from vanna.base import VannaBase
+from vanna.qdrant import Qdrant_VectorStore
+# from qdrant_client import QdrantClient
+from langchain_openai import ChatOpenAI
+
+
+
+class IOIntelligence(VannaBase):
+  def __init__(self, config={}):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)